index.js

```diff
@@ -0,0 +1,2 @@
+module.exports = require('./src/VegaWrapper.js');
```

Comment:
another thing -- in this declaration, you nest v4 inside the old vega wrapper.  Instead I think it should be this:
```
module.exports = {
  wrapper1: require('./src/VegaWrapper.js'),
  wrapper2: require('./src/Vega4Wrapper.js'),
};